mmv1/products/notebooks/api.yaml

```diff
@@ -462,6 +466,480 @@ objects:
+                    description: |
+                      Accelerator model.
+                    values:
+                      - ACCELERATOR_TYPE_UNSPECIFIED
+                      - NVIDIA_TESLA_K80
+                      - NVIDIA_TESLA_P100
+                      - NVIDIA_TESLA_V100
+                      - NVIDIA_TESLA_P4
+                      - NVIDIA_TESLA_T4
+                      - NVIDIA_TESLA_A100
+                  - !ruby/object:Api::Type::Integer
+                    name: "coreCount"
+                    description: |
+                      Count of cores of this accelerator.
+              - !ruby/object:Api::Type::String
+                name: "network"
+                description: |
+                  The Compute Engine network to be used for machine communications.
+                  Cannot be specified with subnetwork. If neither `network` nor
+                  `subnet` is specified, the "default" network of the project is
+                  used, if it exists. A full URL or partial URI. Examples:
+                    * `https://www.googleapis.com/compute/v1/projects/[project_id]/
+                    regions/global/default`
+                    * `projects/[project_id]/regions/global/default`
+                  Runtimes are managed resources inside Google Infrastructure.
+                  Runtimes support the following network configurations:
+                    * Google Managed Network (Network & subnet are empty)
+                    * Consumer Project VPC (network & subnet are required). Requires
+                    configuring Private Service Access.
+                    * Shared VPC (network & subnet are required). Requires
+                    configuring Private Service Access.
+              - !ruby/object:Api::Type::String
+                name: "subnet"
+                description: |
+                  The Compute Engine subnetwork to be used for machine
+                  communications. Cannot be specified with network. A full URL or
+                  partial URI are valid. Examples:
+                    * `https://www.googleapis.com/compute/v1/projects/[project_id]/
+                    regions/us-east1/subnetworks/sub0`
+                    * `projects/[project_id]/regions/us-east1/subnetworks/sub0`
+              - !ruby/object:Api::Type::Boolean
+                name: "internalIpOnly"
+                description: |
+                  If true, runtime will only have internal IP addresses. By default,
+                  runtimes are not restricted to internal IP addresses, and will
+                  have ephemeral external IP addresses assigned to each vm. This
+                  `internal_ip_only` restriction can only be enabled for subnetwork
+                  enabled networks, and all dependencies must be configured to be
+                  accessible without external IP addresses.
+              - !ruby/object:Api::Type::String
+                name: "serviceAccount"
```

Comment:
This field is not on the documentation?

https://cloud.google.com/vertex-ai/docs/workbench/reference/rest/v1/projects.locations.runtimes#VirtualMachineConfig